src/script/page/LeftSidebar/LeftSidebar.tsx

```diff
@@ -69,11 +67,7 @@ const LeftSidebar: React.FC<LeftSidebarProps> = ({listViewModel, selfUser, isAct
   }, []);
 
   return (
-    <div id="left-column" className={cx('left-column', {'left-column--light-theme': !isActivatedAccount})}>
-      <header>
-        <h1 className="visually-hidden">{t('accessibility.headings.sidebar')}</h1>
```

Comment:
for other reviewers: this didn't get removed but moved someplace else. I was a bit confused